nengo/config.py

```diff
@@ -84,6 +84,10 @@ def __repr__(self):
         return "<%s[%s]{%s}>" % (type(self).__name__,
                                  self._configures.__name__, ", ".join(params))
 
+    def __setstate__(self, state):
+        for k, v in state.items():
+            setattr(self, k, v)
```

Comment:
I'll push it this afternoon, but yeah, with that check in `__getattr__` it really just amounts to adding a `__slots__` class attribute. We have to implement `__getstate__` and `__setstate__` regardless.